### DISCUSSION TITLE
How to install LIBXC enabled Quantum Espresso latest build?

### DISCUSSION THREAD

**Discussion by rsielguea** [spack/spack#30050] [Q&A]
Hello,
I am trying to install the latest version of QE (7.0) with LIBXC enabled. Any advice? I am new to Spack.

Thank you

**Comment by cessenat** [comment]
Maybe you can try the "SIRIUS enabled fork of QuantumESPRESSO":
`$ spack install q-e-sirius+libxc`
If you want a newer version, try:
```
$ spack versions q-e-sirius
==> Safe versions (already checksummed):
  develop-ristretto  6.7-rc1-sirius
==> Remote versions (not yet checksummed):
  7.0-rc1-sirius  6.5-rc6-sirius  6.5-rc5-sirius  6.5-rc4-sirius  6.5-rc3-sirius  6.5-rc2-sirius  6.5-rc1-sirius  6.4-rc3-sirius  6.4-rc2-sirius

```
and if 7.0-rc1 is OK you can add the result of
```
$ spack checksum q-e-sirius@7.0-rc1-sirius
    version('7.0-rc1-sirius', sha256='2a3977247140621baad2d97be5c6a44f0f34d076d93b71325076f4a06d29e6c3')
```
above the current 
`    version('6.7-rc1-sirius', tag='v6.7-rc1-sirius', submodules=True)`
line.
Then do
`$ spack install q-e-sirius@7.0-rc1-sirius`

If you are a totally newbee to spack, after git clone and source setup-env.sh do a "spack compiler find".
If lost: https://spack.readthedocs.io/en/latest/getting_started.html
